pixart.js

```diff
@@ -1 +1,50 @@
+// INITIALIZE
+var colorInput = document.querySelector( "#color-field" );
+var colorButton = document.querySelector( "#set-color" );
+var brushBox = document.querySelector( ".brush" );
+var squares = document.getElementsByClassName( "square" );
+var swatches = document.getElementsByClassName( "swatch" );
 
+
+// FUNCTIONS
+var createSwatches = function( number ) {
+  for ( var i=0; i < number; i++ ) {
+    swatch = document.createElement( "div" );
+    swatch.classList.add( "swatch" );
+    document.body.appendChild( swatch );
+  }
+}
+
+var changeBrushBox = function( event ) {
+  event.preventDefault();
+  swatches[2].style.backgroundColor = swatches[1].style.backgroundColor;
+  swatches[1].style.backgroundColor = swatches[0].style.backgroundColor;
+  swatches[0].style.backgroundColor = brushBox.style.backgroundColor;
```

Comment:
Can you think of a way to have an unlimited amount of swatches? i.e. what would you have to change to allow the user to add as many swatches as they like.